klog.go

```diff
@@ -831,6 +831,7 @@ func SetLogger(logr logr.Logger) {
 func SetOutput(w io.Writer) {
 	logging.mu.Lock()
 	defer logging.mu.Unlock()
+	logging.toStderr = false
```

Comment:
SetOutput implies, nothing should go to stderr !!

Reply:
is this a breaking change?

Reply:
more like unspecified behavior i think. example how folks have work around when all they want to do is discard the logs ...
https://github.com/cri-o/cri-o/blob/master/cmd/crio/main.go#L103-L110